docs/developer/eventing/sinks/README.md

```diff
@@ -0,0 +1,83 @@
+# Event sinks
+
+When you create an event source, you can specify a _sink_ where events are sent to from the source. A sink is an Addressable resource that can receive incoming events from other Knative Eventing resources. Knative Services, Channels, and Brokers are all examples of sinks.
```

Comment:
technically speaking, any resources can send events to a sink, not only Knative ones.